envfile/tool.gpt

```diff
@@ -1,5 +1,6 @@
 Name: environment manager
 Share Tools: env
+Type: context
 ---
 Name: env
 Description: Manage the runtime environment variables
```

Comment:
I didn't add `Type: context` to this one because it doesn't look like a context tool. It is only shared from a context tool.